### ISSUE TITLE
Drop downs not populating after upgrade to 1.9.3

### ISSUE THREAD

**Issue by belzebud** [digitalocean/netbox#1000]
Issue type: Bug report

**Python version:**
Python 2.7.9-1
**NetBox version:**
NetBox 1.9.3

I upgraded from NetBox 1.8.3 to 1.9.3 today.  To upgrade I used Option B: Clone the Git Repository. 
 Everything was working fine before the upgrade, but afterwards I noticed that some drop-down menus are not populating any more.

I added a new rack.  I then tried adding a new PDU and linking a PDU socket with a plug on a device but after going to the select tab and choosing the rack (I only have 1 site currently), the device box never populates.  The dashes go away but it never shows a list of devices no matter which rack I select (pic attached).

I also tried linking a new switch port to an ethernet port on a network device and I get the same thing.  If I go back to racks and devices I entered before the upgrade I see the same behavior where I can't link ports/sockets, so it's not just new additions.

This happens no matter where I start the linking process from; the PDU. device, or switch.

When using the search tab to connect ports/sockets the port box never populates so I can't do it that way either.
![netboxdropd](https://cloud.githubusercontent.com/assets/17045516/24360557/f393a084-1307-11e7-9f1f-3d0f6514b088.png)

**Comment by jeremystretch** [comment]
Did you try forcing a refresh in your browser to pull down the updated Javascript?

**Comment by belzebud** [comment]
Sorry, the issue must be with my browser, I've been using Chrome Canary while doing most of the editing.  I haven't let it upgrade or restart it for over a week, so I didn't think to check a different browser.  This morning I tried FireFox and it works there.

I did try to refresh the browser and clear offline data yesterday, but sometimes when I have weird issues with the Canary build it requires a reinstall.
